website_remove_powered_by_odoo/README.rst

```diff
@@ -6,51 +6,8 @@
 Website Remove Powered by Odoo
 ==============================
 
-* This module extends the functionality of website and allows you to remove Powered by Odoo from the website.
-
-
-Installation
-============
-
-To install this module, you need to:
-
- * have basic modules installed (website)
-
-
-Known issues / Roadmap
-======================
-
-* Note: This module inherits the template from the website_sale. It may have conflict with certain themes.
+* This module allows you to remove Powered by Odoo from the website.
+* Note: This module interits the template from the webstie_sale. It may has conflict with certain themes.
```

Comment:
Note: This module inherits the template from the website_sale. It may have conflict with certain themes.